sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -520,25 +520,24 @@ async def connect(self):
                     ssl=self.sslopts,
                     heartbeat=DEFAULT_WEBSOCKET_HEARTBEAT_SECONDS,
                 )
-            except ClientConnectorError as exc:
+            except ClientConnectionError as exc:
```

Comment:
What's the difference between the Connection and the Connector errors? Do we want to handle them both?